src/java.base/share/classes/java/util/Map.java

```diff
@@ -393,9 +393,9 @@
     Set<Map.Entry<K, V>> entrySet();
 
     /**
-     * A map entry (key-value pair). The entry may be unmodifiable, or the
+     * A map entry (key-value pair). The Entry may be unmodifiable, or the
```

Comment:
In that case then maybe it should be `{@code Entry}` in both places where `entry` was replaced with `Entry` ?

Reply:
Maybe. We're not terribly consistent about this. A fair amount of the docs just uses a plain-text, capitalized form of a class name, as opposed to the code font. Using the code font everywhere adds clutter to both the markup and to the rendered output. In this case I wanted to distinguish the generic mention of an entry in a map from an instance of an Entry object. In cases where it needs to be absolutely clear, I used `Map.Entry` (the qualified name, in code font). Doing that here seems like it would add too much clutter though.